packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -661,6 +665,29 @@ export class CommonWell {
     return documentQueryResponseSchema.parse(res.data);
   }
 
+  /**
+   * Retrieves a Document and store it on the local file system.
```

Comment:
Is this true? Can't the WriteStream be to something else other than the local fs?

Reply:
Yep, outdated comment. Good catch!